### ISSUE TITLE
Feature: profiled lens correction

### ISSUE THREAD

**Issue by ghost** [hanatos/vkdt#72]
Could be used in the lens correction node. Could use Lensfun or built in metadata

**Comment by hanatos** [comment]
good idea. probably a matter of fitting/matching the coefficients. i really don't feel like implementing the lensfun/any other formulas if it can be avoided.
